packages/pkp-base/src/lib/pkp-base.ts

```diff
@@ -358,12 +364,7 @@ export class PKPBase<T = PKPBaseDefaultParams> {
       this.throwError('pkpPubKey (aka. uncompressPubKey) is required');
     }
 
-    this.validateAuthContext();
```

Comment:
Can we really get rid of this call and assume everything is correct? It seems like we would still want to call it, _after_ calling `authenticate`?  And if we're going to do that, wouldn't we want to leave the logic _inside_ `validateAuthContext()` intact rather than removing the check for `this.controllerSessionSigs`?

Reply:
ah yes this makes much more sense. Ill update to check after the variable assignment. good catch!

Reply:
Update: I have added a new `validateAuthenticate()` method to check on the internal properties of the object after `getSessionSigs` is called on the `client` which will perform the needed checks similar to `validateAuthContext` but is specific to the checks needed after we generate a new session instance.